src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -747,3 +747,140 @@
+    text: >
+        az acr scope-map list -r myRegistry
+"""
+
+helps['acr token'] = """
+type: group
+short-summary: Manage tokens for an Azure Container Registry.
+"""
+
+helps['acr token create'] = """
+type: command
+short-summary: Creates a token for an Azure Container Registry.
+examples:
+  - name: Create a token associated to the scope map 'myScopeMap'.
+    text: >
+        az acr token create -r myRegistry -n myToken --scope-map myScopeMap
+"""
+
+helps['acr token delete'] = """
+type: command
+short-summary: Deletes a token under an Azure Container Registry.
+examples:
+  - name: Delete the token 'myToken'.
+    text: >
+        az acr token delete -r myRegistry -n myToken
+"""
+
+helps['acr token update'] = """
+type: command
+short-summary: Updates a token under an Azure Container Registry, replacing the scope map associated to it.
+examples:
+  - name: Updates the token 'myToken', making it associated with the scope map 'myScopeMap'.
+    text: >
+        az acr token update -r myRegistry -n myToken --scope-map myScopeMap
+"""
+
+helps['acr token show'] = """
+type: command
+short-summary: Shows details and attributes for a token under an Azure Container Registry.
+examples:
+  - name: Get information for the token 'myToken'.
+    text: >
+        az acr token show -r myRegistry -n myToken
+"""
+
+helps['acr token list'] = """
+type: command
+short-summary: Lists all tokens under an Azure Container Registry.
+examples:
+  - name: List tokens under the registry 'myRegistry'.
+    text: >
+        az acr token list -r myRegistry
+"""
+
+helps['acr token credential'] = """
+type: group
+short-summary: Manage credentials entities for a token under an Azure Container Registry.
+"""
+
+helps['acr token credential generate'] = """
+type: command
+short-summary: Generates both passwords for a token under an Azure Container Registry, unless one password is specified. Replaces old values if they existed. If expiry is not specified, it will be defaulted to the max value.
+examples:
+  - name: Generates passowrd1 for the token 'myToken', with expiration for 2 years.
+    text: az acr token credential generate -r myRegistry -n myToken --password1 --years 2
+"""
+
+helps['acr token credential add-certificate'] = """
+type: command
+short-summary: Adds a certificate in a PEM format for a token under an Azure Container Registry. If target certificate already existed, it will be replaced with the new one.
+examples:
+  - name: Adds certificate1 for the token 'myToken'.
+    text: az acr token credential add-certificate -r myRegistry -n myToken --cert1 C:\\myCertificates\\certificate.pem
+"""
+
+helps['acr token credential delete'] = """
+type: command
+short-summary: Reset a token credential. To be used upon expiration, loss or leak of information.
+examples:
+  - name: Reset credentials for the token 'myToken', setting duration of 2 years.
+    text: az acr token credential reset -r myRegistry -n myToken --years 2
```

Comment:
What is this "az acr token credential reset" ?

Reply:
Sorry, I was working with this on a first moment, forgot to remove all residues.

Reply:
Removed.